.travis.yml

```diff
@@ -210,6 +210,11 @@ script:
   - date > time.txt && curl -v --fail -T time.txt ftp://localhost:8021 --user $GALAXY_USER:$GALAXY_USER_PASSWD || true
   # Test FTP Server get
   - curl -v --fail ftp://localhost:8021 --user $GALAXY_USER:$GALAXY_USER_PASSWD
+  
+  # Test CVMFS
+  - docker_exec_run bash -c "service autofs start"
+  - docker_exec_run bash -c "cvmfs_config chksetup"
```

Comment:
This is for the compose jobs, and it can only be because the install steps aren't getting run. Any idea why that is? Where is the ansible command for those steps?